### DISCUSSION TITLE
Problem of sequential metabolism PBPK model

### DISCUSSION THREAD

**Discussion by mmalsmadi** [Open-Systems-Pharmacology/Forum#1582] [Questions & Problems]
Hello 

I am trying to simulate the PK of the parent compound (methadone R and S stereoisomers) and their common metabolite (EDDP) after IV-bolus and oral administration

The parent compound simulate fine, but the metabolite (EDDP) doesn't show even previous work showed possible similar simulation using PK-Sim
https://pubmed.ncbi.nlm.nih.gov/35491971/

Any thoughts on what causes this issue

P.S. The PK-Sim simulation file is attached here.
[Methadone & EDDP.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/12228336/Methadone.EDDP.zip)

Thanks

**Comment by Yuri05** [comment]
seems something is missing in your model.
E.g. looking at the EDDP venous blood concentration in the 1st simulation: it's const zero in your simulation, while in the paper it's apparently not. 
<img width="500" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/93a9b356-62ed-4f15-8f22-2c0d70ceb8d8">  <img width="300" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/52ea7eb1-d801-4a77-9956-14f15637ae26">

**Reply by Yuri05** [reply]
All enzymes in your project have relative expressions=0 everywhere, that's why
![grafik](https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/593c79ad-b3a4-4508-871c-7adb63f87e50)
